suite-common/wallet-core/src/discovery/discoveryThunks.ts

```diff
@@ -134,7 +134,7 @@ const handleProgressThunk = createThunk(
             }),
         );
 
-        // change auth confirm field only if metadata are not enabled
+        // change auth confirm field only if metadata is not enabled
```

Comment:
It's quite a linguistic conundrum 😆 I believe "are" was more correct here, but of course it doesn't matter here...

Reply:
removed